Update Qene and PoemQene

CarstenHoffmannMarburg opened the pull request:
I updated these two authority files for Qene according to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2853#issuecomment-2689050538.

CarstenHoffmannMarburg committed:
Update Qene and PoemQene

I updated these two authority files for Qene according to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2853#issuecomment-2689050538.

CarstenHoffmannMarburg committed:
minor corrects

karljonaskarlsson commented on Additions/PoemQene.xml:
```diff
@@ -41,8 +41,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <p>This keyword is used to classify additiones.</p>
-                <p>Despite wide-spread opinion that qǝne is a poetic improvisation which is not written down, manuscripts contain unexpectedly many qǝne-poems, usually as additional notes. 
-                    Frequently it is clearly stated the the poem is qǝne, and even the type is indicated. But no one tried to check if these poems formally correspond to what we know today as qǝne-forms. </p>
+                <p>Despite wide-spread opinion that qǝne is a poetic improvisation which is not written down, manuscripts contain unexpectedly many qǝne-poems, for example as additional notes. 
```
"the wide-spread"

DenisNosnitsin1970 replied:
I would say, manuscripts contain "a number of ...". We picked many, but it happened thanks to two manuscripts.

karljonaskarlsson commented on Genres/Qene.xml:
```diff
@@ -33,9 +33,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                <p>Despite wide-spread opinion that qǝne is a poetic improvisation which is not written down, manuscripts contain unexpectedly many qǝne-poems, usually, but not only as additional notes. 
-                    Frequently it is clearly stated the the poem is qǝne, and even the type is indicated. But no one tried to check if these poems formally correspond to what we know today as qǝne-forms. </p>
-                <p>Wherever possible, we mark qǝne-forms with this specific keyword; when it is indicated, we also specify the specific modes, see <ref type="authFile" corresp="EtanaMogar">ʿəṭāna mogar</ref>,  <ref type="authFile" corresp="Mawaddes">mawaddəs</ref></p>
+                <p>Despite wide-spread opinion that qǝne is a poetic improvisation which is not written down, manuscripts contain unexpectedly many of them in additional notes or in collections of qǝne-poems. 
```
"the wide-spread"

karljonaskarlsson commented on Genres/Qene.xml:
```diff
@@ -33,9 +33,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                <p>Wherever possible, we mark qǝne-forms with this specific keyword; when it is indicated, we also specify the specific modes, see <ref type="authFile" corresp="EtanaMogar">ʿəṭāna mogar</ref>,  <ref type="authFile" corresp="Mawaddes">mawaddəs</ref></p>
+                <p>Despite wide-spread opinion that qǝne is a poetic improvisation which is not written down, manuscripts contain unexpectedly many of them in additional notes or in collections of qǝne-poems. 
+                    Frequently it is clearly stated the poem is qǝne, and even the type is indicated.</p>
+                <p>Wherever possible, we define qǝne-forms according to the classification 
```
"the classification of" or "the classification found in" or something similar

karljonaskarlsson commented on Genres/Qene.xml:
```diff
@@ -33,9 +33,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                <p>Despite wide-spread opinion that qǝne is a poetic improvisation which is not written down, manuscripts contain unexpectedly many of them in additional notes or in collections of qǝne-poems. 
+                    Frequently it is clearly stated the poem is qǝne, and even the type is indicated.</p>
+                <p>Wherever possible, we define qǝne-forms according to the classification 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl> and mark it with this specific keyword and
```
This sentence seems to be unfinished.

CarstenHoffmannMarburg replied:
Yes.

karljonaskarlsson commented on Genres/Qene.xml:
```diff
@@ -33,9 +33,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                <p>Wherever possible, we define qǝne-forms according to the classification 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl> and mark it with this specific keyword and
+                    Cf. 
+                    <ref type="authFile" corresp="EtanaMogar"/>,  
```
I guess there should be a comma after each of these keywords.

Maybe they could be sorted alphabetically?

DenisNosnitsin1970 replied:
Possible; but Guidi arranged them according to the length (number of verses, starting from the guba'e qana of two verses). As step two, we can include short info on this subject, to each of the type.

karljonaskarlsson replied:
If it's made clear that this is the logic behind the sorting, it would also work well

CarstenHoffmannMarburg replied:
I think with the number of verses in brackets it is clear enough what the logic behind the sorting is. Do you agree?

karljonaskarlsson reviewed (commented):
Good updates! Thank you!

CarstenHoffmannMarburg committed:
Up according to Jonas recommendations

DenisNosnitsin1970 commented on Genres/Qene.xml:
```diff
@@ -33,9 +33,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                    Frequently it is clearly stated the the poem is qǝne, and even the type is indicated. But no one tried to check if these poems formally correspond to what we know today as qǝne-forms. </p>
-                <p>Wherever possible, we mark qǝne-forms with this specific keyword; when it is indicated, we also specify the specific modes, see <ref type="authFile" corresp="EtanaMogar">ʿəṭāna mogar</ref>,  <ref type="authFile" corresp="Mawaddes">mawaddəs</ref></p>
+                <p>Despite wide-spread opinion that qǝne is a poetic improvisation which is not written down, manuscripts contain unexpectedly many of them in additional notes or in collections of qǝne-poems. 
+                    Frequently it is clearly stated the poem is qǝne, and even the type is indicated.</p>
```
"and the type and the author are indicated".

DenisNosnitsin1970 commented on Genres/Qene.xml:
```diff
@@ -33,9 +33,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                    <ref type="authFile" corresp="GubaeQana"/>
+                    <ref type="authFile" corresp="KebrYeeti"/>
+                    <ref type="authFile" corresp="Kwellekomu"/>
+                    <ref type="authFile" corresp="Mawaddes"/>
```
Mawaddes is mentioned twice?

CarstenHoffmannMarburg commented:
> Possible; but Guidi arranged them according to the length (number of verses, starting from the guba'e qana of two verses). As step two, we can include short info on this subject, to each of the type.

I would like to add the number of verses and order the qene types according to the number of verses. However, I do not see any keyword for ʾamlākiyā or za-ʾamlākiyā-type of qəne with three verses. Is it possibly, that qəne-poems with three verses are frequently called śəllāse and that it is only Guidi, who has the opinion that these poems are named ʾamlākiyā or za-ʾamlākiyā? Do we need a keyword for ʾamlākiyā or za-ʾamlākiyā or shall we note the ambivalence in the authFile for śəllāse?

I think, in a second step, also the authority files for the qene could be revised.

CarstenHoffmannMarburg commented:
> However, I do not see any keyword for ʾamlākiyā or za-ʾamlākiyā-type of qəne with three verses. 

We do not have a keyword for ʾačǝr kʷǝllǝkǝmu and ʾačər wāzemā (two verses) either.

CarstenHoffmannMarburg commented:
> As step two, we can include short info on this subject, to each of the type.

Having added the number of verses for each type in brackets, I would also like to enhance the record with an attribute for each mode. Would you agree to classify ʾarārāy as "joyous", ʿəzl as "solemn" and gəʿz as "melancholic"?

CarstenHoffmannMarburg committed:
Denis requests + qene types and modes

CarstenHoffmannMarburg committed:
Texts in div in PoemQene

DenisNosnitsin1970 commented:
"Sellase" is six verses, this is correct and I saw it in other sources, "three" is a modern interpretaion (which I think is misled). So far, possibly only those types already in the Bm which occured at least once in the manuscripts catalogued in Bm. There has been no single example for others, and no need in keywords etc. This fact also means something. I am indecisive as to if we should include all, I see pros and contras. (I personally tend to limit the arrangement of our records for those phenomena only which we truly encounter in manucripts, since this is the reality (in other words, I would include za-amlakiya whe we have found an exaple), but I do not insist and it can be done differently).
 " Would you agree to classify ʾarārāy as "joyous", ʿəzl as "solemn" and gəʿz as "melancholic"?" - ok, this is reasonable, but now I do not know if they apply to all types (if I remember well, not to all). Also, do they coordinate with the modes in use for chants? Smth might be already in the Bm.

CarstenHoffmannMarburg commented:
> "Sellase" is six verses, this is correct and I saw it in other sources, "three" is a modern interpretaion (which I think is misled). So far, possibly only those types already in the Bm which occured at least once in the manuscripts catalogued in Bm. There has been no single example for others, and no need in keywords etc. This fact also means something. I am indecisive as to if we should include all, I see pros and contras. (I personally tend to limit the arrangement of our records for those phenomena only which we truly encounter in manucripts, since this is the reality (in other words, I would include za-amlakiya whe we have found an exaple), but I do not insist and it can be done differently). " Would you agree to classify ʾarārāy as "joyous", ʿəzl as "solemn" and gəʿz as "melancholic"?" - ok, this is reasonable, but now I do not know if they apply to all types (if I remember well, not to all). Also, do they coordinate with the modes in use for chants? Smth might be already in the Bm.

I agree with your opinion to maintain the current strategy. I will update the record once again to make the ambivalences a little more clear. What you said about sellase is one more reason to update the records on the qene types, as well. The modes as encountered in Bm refer both to chants and poems as far as I know. They are not used very often, as far as I see.

CarstenHoffmannMarburg committed:
Update ESmma008 (minor corrects)

CarstenHoffmannMarburg committed:
Update Petros in ESmhg004

karljonaskarlsson commented:
> > As step two, we can include short info on this subject, to each of the type.
> 
> Having added the number of verses for each type in brackets, I would also like to enhance the record with an attribute for each mode. Would you agree to classify ʾarārāy as "joyous", ʿəzl as "solemn" and gəʿz as "melancholic"?

No, I think it can only be misleading to equate the modes with such adjectives. Why not simply state which mode they are in?

CarstenHoffmannMarburg committed:
Up Qene (Sellase, Gez-mode)

According to Denis request.

CarstenHoffmannMarburg committed:
Delete attributes of modes

According to Jonas request.

CarstenHoffmannMarburg committed:
Del poem from ESmma008

Because of ambivalence regarding the type of the poem, which is indicated as Etana Mogar (seven verses) and the fact, that there are eight verses discernible (cf. https://github.com/BetaMasaheft/Works/pull/1478/files), I decided to delete this as an example.

CarstenHoffmannMarburg committed:
Up according to Denis request

CarstenHoffmannMarburg committed:
Update order of qene types

according to Denis and Jonas suggestion.

CarstenHoffmannMarburg commented:
@DenisNosnitsin1970, I think, I have solved all problems. Can you approve this pull request?
